Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -165,6 +219,71 @@ def set_default_job_id(client: Client, args: Dict[str, Any]) -> CommandResults:
 ''' MAIN FUNCTION '''
 
 
+def send_event_feedback(client: Client, feature_mapping_field: str, args: Dict[str, Any]) -> CommandResults:
+    job_id = args.get("job_id", None)
+    if not job_id:
+        job_id = client.get_default_job_id()
+    event_id = args.get("event_id")
+    mappings = parse_mappings(feature_mapping_field)
+
+    username = args.get("username")
+    label = args.get("label")
+    closing_notes = args.get("closing_notes", "")
+    indicators = args.get("indicators", None)
+    arcanna_label = mappings.get(label, None)
+    if arcanna_label is None:
+        demisto.log(f"Error unknown label supplied.label={label}")
+        raise Exception(f"Error in arcanna-send-feedback.Wrong label")
+
+    response = client.send_feedback(job_id, event_id, username, arcanna_label, closing_notes, indicators)
+    readable_output = f'## {response}'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Arcanna.Feedback',
+        outputs_key_field='status',
+        outputs=response
+    )
+
+
+def send_bulk_events(client: Client, feature_mapping_field: str, args: Dict[str, Any]) -> dict:
+    job_id = args.get("job_id", None)
+    if not job_id:
+        job_id = client.get_default_job_id()
+    events = args.get("events")
```

Comment:
Please use `argToList` to convert str to a comma-separeted list.
```suggestion
    events = argToList(args.get("events"))
```

Reply:
Done